custom_components/tuya_local/devices/Fresco_Hydrate_Ultra_Pet_Fountain.yaml

```diff
@@ -0,0 +1,111 @@
+name: Hydrate Ultra
+products:
+  - id: cuw2k3ho3e25fade
+    name: Hydrate Ultra
+
+primary_entity:
+  entity: light
+  icon: "mdi:water-pump"
+  dps:
+    - id: 101
+      type: boolean
+      name: switch
+    - id: 102
+      name: brightness
+      type: integer
+      range:
+        min: 1
+        max: 10
+
+secondary_entities:
+  - entity: button
+    name: Reset Filter Alarm
+    icon: "mdi:air-filter"
+    category: config
+    dps:
+      - id: 104
+        type: integer
+        name: button
+  - entity: sensor
+    name: Filter life
+    icon: "mdi:air-filter"
+    category: diagnostic
+    dps:
+      - id: 104
+        type: integer
+        name: sensor
+        precision: 0
+        range:
+          min: 0
+          max: 43200
+        mapping:
+          - scale: 1440
```

Comment:
Instead of scaling this to days, add `class: duration` to the entity config, and change the unit to `min`.  Then HA should scale as appropriate.

Reply:
Why would i do that ?
Its working correct, it is based on days not minutes... 
So when press the button it resets to 30 days for water and 60 days for filter

Reply:
> Instead of scaling this to days, add `class: duration` to the entity config, and change the unit to `min`. Then HA should scale as appropriate.

That doesn't work i did try that, it shows up like this:
```
Filter life
37,462 min
Pump maintenance
80,662 min
```

example:

```
  - entity: sensor
    name: Filter life
    icon: "mdi:air-filter"
    category: diagnostic
    dps:
      - id: 104
        type: integer
        name: sensor
        precision: 0
        range:
          min: 0
          max: 43200
        mapping:
          - invert: true
        unit: min
        class: duration
```

Reply:
You did not add `class: duration` to the entity config.

Reply:
Example:
```
  - entity: sensor
    name: Filter life
    icon: "mdi:air-filter"
    category: diagnostic
    class: duration
    dps:
      - id: 104
        type: integer
        name: sensor
        range:
          min: 0
          max: 43200
        mapping:
          - invert: true
        unit: min
```